governance.md

```diff
@@ -0,0 +1,149 @@
+# Governance
+
+Coraza is an open-source WAF solution compatible with seclang and CoreRuleset.
+
+The Coraza project aims for open and transparent governance and decision-making, thus encouraging community building and contribution.
+
+A formal governance structure helps us to
+
+- Provide a structure for individuals to become involved in the project.
+- Communicate all processes for members to operate within the project.
+- Document a system for open product development, roadmapping, and planning.
+- Provide a means for making decisions if consensus cannot be reached.
+
+This document describes the rules and governance of the Coraza project. It is meant to be followed by all the developers of the project and the Coraza community.
+
+## Community overview
```

Comment:
Just a share you may have already read it but if not, GitHub's guide is really awesome. Perhaps there are tidbits you can source if you haven't seen it yet

https://opensource.guide/building-community/

Reply:
Agreed on this.